HaxeManual/02-types.tex

```diff
@@ -808,7 +808,7 @@ \subsection{Implicit Casts}
 Class field casts have the same semantics, but are defined completely differently:
 
 \haxe{assets/ImplicitCastField.hx}
-By adding \expr{@:from} to a static function, that function qualifies as implicit cast function from its argument type to the abstract. These functions must return a value of the abstract type. They must also be declared \expr{static}.
+By adding \expr{@:from} to a static function, that function qualifies as implicit cast function from its argument type to the abstract. These functions must return the value of the abstract type. They must also be declared \expr{static}.
```

Comment:
No, they return *a* value that is of the abstract type, i.e. a new instance of that abstract.